sql/core/src/main/scala/org/apache/spark/sql/sources/interfaces.scala

```diff
@@ -422,7 +422,7 @@ abstract class HadoopFsRelation private[sql](
     parameters: Map[String, String])
   extends BaseRelation with FileRelation with Logging {
 
-  override def toString: String = getClass.getSimpleName + paths.mkString("[", ",", "]")
+  override def toString: String = getClass.getSimpleName
```

Comment:
Removed the paths string because it's now shown as part of the metadata in both `simpleString` and visualized plan node.